docs/content/ProgressBar.mdx

```diff
@@ -2,32 +2,32 @@
 title: ProgressBar
 ---
 
-
-Use `ProgressBar` to visualize task completion. 
+Use `ProgressBar` to visualize task completion.
 
 ## Default example
 
 ```jsx live
-<ProgressBar progress={80}/>
- ```
+<ProgressBar progress={80} />
+```
+
+If you'd like to use ProgressBar inline, pass the `inline` boolean prop & **be sure to set a width**.
 
- If you'd like to use ProgressBar inline, pass the `inline` boolean prop & **be sure to set a width**.
- ```jsx live
-  <>
-    <Text mr={3}>5 of 10</Text>
-    <ProgressBar progress={50} inline width='100px'/>
-  </>
- ```
+```jsx live
+<>
+  <Text mr={3}>5 of 10</Text>
+  <ProgressBar progress={50} inline width="100px" />
+</>
+```
 
 ## System props
 
 ProgressBar components get `COMMON` system props. Read our [System Props](/system-props) doc page for a full list of available props.
 
 ## Component props
 
-| Name | Type | Default | Description |
-| :- | :- | :-: | :- |
-| progress | Number | | Used to set the size of the green bar |
-| barSize | String | 'default' | Controls the height of the progress bar. Can be 'small', 'large', or 'default'. If omitted, height is set to the default height. |
-| inline | Boolean | false | Styles the progress bar inline |
-| bg | String | 'green.5' | Set the progress bar color, defaults to bg-green |
+| Name     | Type    |     Default     | Description                                                                                                                      |
+| :------- | :------ | :-------------: | :------------------------------------------------------------------------------------------------------------------------------- |
+| progress | Number  |                 | Used to set the size of the green bar                                                                                            |
+| barSize  | String  |    'default'    | Controls the height of the progress bar. Can be 'small', 'large', or 'default'. If omitted, height is set to the default height. |
+| inline   | Boolean |      false      | Styles the progress bar inline                                                                                                   |
+| bg       | String  | 'state.success' | Set the progress bar color, defaults to bg-green                                                                                 |
```

Comment:
I just noticed that `state.success` is not a color that's defined in Primer Primitives. Let's see if we can find an alternative.

Reply:
`bg.successInverse` might be the closest?

Reply:
![image](https://user-images.githubusercontent.com/16739070/110381007-885b6f80-801e-11eb-8dc5-4b20e621e20d.png)

yep, looks good to me!